docs/guide/apm-getting-started.asciidoc

```diff
@@ -1,4 +1,7 @@
-[[apm-getting-started]]
+:branch: current
+include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
```

Comment:
I wonder why we need this in both files? I would assume in the `index.asciidoc` header should be enough?

Reply:
@watson Ignore this, just saw the conversation with dede that `guide` is a different book.